core/dbt/deprecations.py

```diff
@@ -84,6 +84,15 @@ class NotADictionaryDeprecation(DBTDeprecation):
     '''.lstrip()
 
 
+class ColumnQuotingDeprecation(DBTDeprecation):
+    _name = 'column-quoting-unset'
+
+    _description = '''
+    The quote_columns parameter was not set for seeds, so the default value of
+    False was chosen. The default will change to True in a future release.
```

Comment:
can you add a link to 
https://docs.getdbt.com/v0.15/docs/seeds#section-specify-column-quoting